common/sgx/revocation.c

```diff
@@ -284,6 +287,42 @@ oe_result_t oe_get_revocation_info_from_certs(
     return result;
 }
 
+/**
```

Comment:
Given the existing functions in this revocation.c are going away, I suggest creating a new file to hold the new functions or code changes that you made into that new file, such as collaterals.c or a better name.




add a new file, instead of putting inside revocationinfo.c --> 
test on the host verifier

Does it run with INtel's dcap client

QVE Identifier info
 Need to take this information


sgx_ql_get_qve_identity

Reply:
Thanks, used collateral.c/collateralinfo.c to hold new functions.